publisher/product/ontology/build.sh

```diff
@@ -523,26 +537,26 @@ function ontologyZipFiles () {
     find . -type f -name \*\.ttl -exec /bin/bash -c 'perl -pi -e "s/^\s*(\@prefix)\s+([^\s]+)\s+(\<[^\>]*\>)\s+(\.)\s*$/\1 \2 \3 \4\n/g" "{}"' \;
 
 
-    ${FIND}  "${family_product_branch_tag}" -name '*.ttl' -print | ${GREP} -v etc |  ${GREP} -v "LoadFIBOProd.ttl" | grep -v About |  xargs zip ${zipttlDevFile}
+    ${FIND}  "${family_product_branch_tag}" -name '*.ttl' -print | ${GREP} -v etc |  ${GREP} -v "Load${ONTPUB_FAMILY}Prod.ttl" | grep -v About |  xargs zip ${zipttlDevFile}
```

Comment:
Should be uppercase:
`${FIND} "${family_product_branch_tag}" -name '*.ttl' -print | ${GREP} -v etc | ${GREP} -v "Load${ONTPUB_FAMILY^^}Prod.ttl" | grep -v About | xargs zip ${zipttlDevFile}`